zfs.html.markdown

```diff
@@ -19,23 +20,24 @@ usability for systems administrators.
 
 ### Virtual Devices
 
-A VDEV is similar to a raid device presented by a RAID card, there are several different
-types of VDEV's that offer various advantages, including redundancy and speed.  In general
-VDEV's offer better reliability and safety than a RAID card.  It is discouraged to use a
-RAID setup with ZFS, as ZFS expects to directly manage the underlying disks.
-
-Types of VDEV's
-
-* mirror (n-way mirrors supported)
-* raidz
-	* raidz1 (1-disk parity, similar to RAID 5)
-	* raidz2 (2-disk parity, similar to RAID 6)
-	* raidz3 (3-disk parity, no RAID analog)
-* disk
-* file (not recommended for production due to another filesystem adding unnecessary layering)
-
-Your data is striped across all the VDEV's present in your Storage Pool, so more VDEV's will
-increase your IOPS.
+A VDEV (Virtual Device) in ZFS is analogous to a RAID device and simmilaly offers different
+benefits in terms of redundancy and performance. In general VDEV's offer better reliability 
+and safety than a RAID card. It is discouraged to use a RAID setup with ZFS, as ZFS expects 
+to directly manage the underlying disks.
+
+| VDEV Type | Similar RAID | Notes |
+|-----------|----------------|---------------------------------------|
+| Stripe | Single disk | No redundancy. Data is not mirrored or parity-protected. |
```

Comment:
I think it should be removed because no mention [here](https://openzfs.org/wiki/System_Administration#Low_level_storage) about `stripe`.